phys2bids/tests/conftest.py

```diff
@@ -1,7 +1,6 @@
 import os
-from urllib.request import urlretrieve
-
 import pytest
```

Comment:
`pytest` is not part of the standard library and should be separate into a third-part imports section.

Reply:
This point still stands! `pytest` is not part of the stdlib and should be moved to a third-party library import section.